appveyor.yml

```diff
@@ -0,0 +1,44 @@
+version: 1.0.{build}
+image:
+  - Visual Studio 2019
+  - Ubuntu
+pull_requests:
+  do_not_increment_build_number: true
+branches:
+  only:
+  - master
+  - dev
+dotnet_csproj:
+  patch: true
+  file: '**\*.csproj'
+  version: '{version}'
+  package_version: '{version}'
+  assembly_version: '{version}'
+  file_version: '{version}'
+  informational_version: '{version}'
+nuget:
+  disable_publish_on_pr: true
+environment:
+  CODECOV_TOKEN: 93f1ba74-2bc4-4b01-8e9e-3a3c97825e91
+build_script:
+- ps: "dotnet pack src/AspNetCore.AutoHealthCheck.sln -o ./artefacts/"
+test_script:
+- ps: "dotnet test src/AspNetCore.AutoHealthCheck.sln"
+artifacts:
+- path: ./artefacts/*.nupkg
+  name: NuGet
+
+# Publish to nuget when running in ubuntu
+for:
+-
+  matrix:
+    only:
+      - image: Ubuntu
+  deploy:
+  - provider: NuGet
+    name: production
+    api_key:
+      secure: "Use 'Encrypt configuration data' page in AppVeyor (Account → Encrypt YAML) to encrypt your nuget API key and place it here"
```

Comment:
@davidrevoledo doesn't look like nuget publishing was previously configured. I could remove this to simplify this pr.